apps/charterafrica/src/pages/api/v1/github/[path].page.js

```diff
@@ -0,0 +1,24 @@
+import processData from "@/charterafrica/lib/tools/processData";
```

Comment:
This should be in` api/v1/resources`. GitHub is just one source of data sources... adding GitLab, etc., shouldn't necessitate adding new end points.